src/main/java/org/spongepowered/api/entity/living/ArmorStand.java

```diff
@@ -132,18 +132,18 @@
     void setSmall(boolean small);
 
     /**
-     * Returns whether this armor stand is rendered invisible.
+     * Returns whether this armor stand is affected by gravity or not.
      *
-     * @return Whether this armor stand is invisible
+     * @return Whether this is affected by gravity or not
```

Comment:
`this armor stand is ...` sounds better than `this is ...`, only `this` would refer to the operation of the method, and not armor stand.